src/controller/CHIPDeviceController.cpp

```diff
@@ -801,6 +822,10 @@ void DeviceCommissioner::PersistDeviceList()
             }
             chip::Platform::MemoryFree(serialized);
         }
+
+        char keyIDStr[kMaxKeyIDStringSize];
+        snprintf(keyIDStr, sizeof(keyIDStr), "%d", mNextKeyId);
+        PERSISTENT_KEY_OP(static_cast<uint64_t>(0), kNextAvailableKeyID, key, mStorageDelegate->SetKeyValue(key, keyIDStr));
```

Comment:
If you removed PERSISTENT_KEY_OP from the loading part, you probably need to remove it here as well to make the keys match, right?

Reply:
Yes, thanks for catching..